.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Images to GHCR
+
+on:
+  push:
+    branches:
+      - add-latest-numbers-app
+  workflow_dispatch:
+
+jobs:
+      push-store-image:
+        runs-on: ubuntu-latest
+        steps:
+          - name: 'Checkout GitHub Action'
+            uses: actions/checkout@main
+
+          - name: 'Login to GitHub Container Registry'
+            uses: docker/login-action@v1
+            with:
+              registry: ghcr.io
+              username: ${{github.actor}}
+              password: ${{secrets.GITHUB_TOKEN}}
+
+          - name: 'Build Inventory Image'
+            run: |
+              docker build . --tag ghcr.io/${{github.actor}}/latestnumbers:latest
+              docker push ghcr.io/${{github.actor}}>/latestnumbers:latest
```

Comment:
I usually use https://github.com/docker/build-push-action , but I guess this works too. The only issue is that it's not version tagged. I like to use https://docs.docker.com/build/ci/github-actions/manage-tags-labels/ , to at least generate a tag based on the git commit SHA, so that you know exactly what code you're deploying, and that a restart doesn't suddenly pull a newer version of the code.